### PR TITLE
Toxic Exposures - Multiple Exposures - populate request with herbicide/otherHerbicideDetails

### PR DESCRIPTION
**Pull request by sethdarragile6**
Transforms Herbicide Hazard details object (and Other Herbicide Details object) into Lighthouse Toxic Exposure multiple exposures array

## Summary

- *This work is behind a feature toggle (flipper): YES* `disability_526_toxic_exposure`
- *(Summarize the changes that have been made to the platform)*
- *(If bug, how to reproduce)*
- *(What is the solution, why is this the solution?)*
- *(Which team do you work for, does your team own the maintenance of this component?)*
Disability Benefits Experience Team 1 (DBEX-TREX) 🦖
- *(If introducing a flipper, what is the success criteria being targeted?)*

## Related issue(s)

- Handle multiple exposure durations - test through BE -15C#[83034](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/83034)
- *Link to previous change of the code/bug (if applicable)*
- *Link to epic if not included in ticket*

## Testing done

- [x] *New code is covered by unit tests*
- *Describe what the old behavior was prior to the change*
- *Describe the steps required to verify your changes are working as expected. Exclusively stating 'Specs run' is NOT acceptable as appropriate testing*
- *If this work is behind a flipper:*
  - *Tests need to be written for both the flipper on and flipper off scenarios. [Docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#Featuretogglesguide-Backendexample).*
  - *What is the testing plan for rolling out the feature?*

## Screenshots
image

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [x]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: spec/support/disability_compensation_form/all_claims_fe_submission.json [hunk 1]
```diff
@@ -152,11 +152,40 @@
...
+        "kuwait": true,
+        "iraq": true,
+        "qatar": true
       },
       "gulfWar2001": {
         "djibouti": true,
-        "lebanon": true
+        "lebanon": true,
+        "uzbekistan": true,
+        "airspace": true
```
Comment (christinec-fftc) [inline spec/support/disability_compensation_form/all_claims_fe_submission.json:163]: FE may also send values with `none` or `notsure`. Is this something that we should include in unit tests?

Comment (sethdarragile6) [reply spec/support/disability_compensation_form/all_claims_fe_submission.json:163]: We're already testing these scenarios [here](https://github.com/department-of-veterans-affairs/vets-api/blob/76c1cd11159e71387ff6c6f813a0e12cd8838d02/spec/lib/evss/disability_compensation_form/form526_to_lighthouse_transform_spec.rb#L293)- not explicitly with herbicide locations, but both gulf war and herbicide run through the same common logic

Comment (christinec-fftc) [reply spec/support/disability_compensation_form/all_claims_fe_submission.json:163]: awesome, thanks!

**Review by freeheeling — APPROVED** [review]
lgtm

**Pull request merged**